src/scrcmd_c.c

```diff
@@ -4038,13 +4038,13 @@ BOOL ScrCmd_571(SCRIPTCONTEXT *ctx) {
     u16 r7 = ScriptGetVar(ctx);
     STRING *r7_str;
     STRING *sp0_str;
-    MessageFormat *msgFmt = MessageFormat_new(32);
```

Comment:
new -> New (may be out of scope)

Reply:
nah you're good always a good idea to fix these when we see em